waiter/lib/travis/web/app.rb

```diff
@@ -221,6 +221,7 @@ def set_config(string, opts = {})
 
       config['defaultTitle'] = title
       config['apiEndpoint'] = options[:api_endpoint] if options[:api_endpoint]
+      config['githubAppsEndpoint'] = options[:source_endpoint] + '/github-apps' if options[:source_endpoint] && !options[:source_endpoint].include? 'github.com'
       config['sourceEndpoint'] = options[:source_endpoint] if options[:source_endpoint]
```

Comment:
Current solution will not set `githubAppsEndpoint` for hosted. I'd like to suggest the following improvement:

```suggestion
      config['githubAppsEndpoint'] = options[:github_apps_endpoint]
      source_endpoint = options[:source_endpoint]
      if source_endpoint
        config['sourceEndpoint'] = source_endpoint
        config['githubAppsEndpoint'] = source_endpoint + '/github-apps' unless source_endpoint.include? 'github.com'
      end
```

How does it look to you?